### DISCUSSION TITLE
SCSS new `@use` syntax

### DISCUSSION THREAD

**Discussion by Sysix** [twbs/bootstrap#33814] [Ideas] [answered]
(Dart) SCSS has the feature called `@use`, which should replace `@import`.

Learn more about it here:
https://sass-lang.com/documentation/at-rules/use

The questions is, when Bootstrap will switch to Dart-SCSS and use this feature?

> The Sass team discourages the continued use of the `@import` rule. Sass will gradually phase it out over the next few years, and eventually remove it from the language entirely. Prefer the `@use` rule instead. (Note that only Dart Sass currently supports `@use`. Users of other implementations must use the `@import` rule instead.)
> https://sass-lang.com/documentation/at-rules/import

**Comment by mdo** [comment, marked as answer]
We've answered this in other discussions and issues, as well as our blog posts. Migrating to `@use` would be too big of a jump in our opinion to make it into v5. We can revisit this for v6.

**Reply by Sysix** [reply]
It was clear for me, that it will not be in v5. But I'm happy to see it in your next big release.
Can you maybe link the issues here? Because I didn't find anything about it in the github search :) Ty anyway.
